src/ImageRunModal.jsx

```diff
@@ -635,6 +638,75 @@
         return owner === systemOwner;
     };
 
+    prefillModal() {
+        const container = this.props.container;
+        const containerDetail = this.props.containerDetail;
+        const image = this.props.localImages.find(img => img.Id === container.ImageID);
+        const owner = container.isSystem ? 'system' : this.props.user;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-111750-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)